velox/functions/prestosql/tests/BitwiseTest.cpp

```diff
@@ -120,6 +126,28 @@ TEST_F(BitwiseTest, bitwiseAnd) {
   EXPECT_EQ(bitwiseAnd<int64_t>(kMin64, -1), kMin64);
 }
 
+TEST_F(BitwiseTest, bitCount) {
+  EXPECT_EQ(bitCount<int64_t>(9, kMaxBits), 2);
+  EXPECT_EQ(bitCount<int64_t>(9, 8), 2);
+  EXPECT_EQ(bitCount<int64_t>(-7, kMaxBits), 62);
+  EXPECT_EQ(bitCount<int64_t>(-7, 8), 6);
```

Comment:
Since bitCount template is used only with a single type, int64_t, it can be a regular method, e.g.

```
  std::optional<int64_t> bitCount(std::optional<int64_t> a, std::optional<int64_t> b) {
    return evaluateOnce<int64_t>("bit_count(c0, c1)", a, b);
  }
```